spec/v1/annotations/object-model.yaml

```diff
@@ -31,6 +31,21 @@ definitions:
         title: Element Reference
         ref: "#/definitions/ElementReference"
 
+  "@ObjectModel.custom":
+    type: boolean
+    description: |-
+      If set to true, it is a custom element (field, entity, service, etc.).
+      If set to false, it is not a custom element.
+      If annotation not present, then it is not defined whether it is a custom element.
+      Note that `@ObjectModel.customer: false` is not the same as if the annotation is not present at all.
+
+      If applied to an entity or service, everything that it contains is also considered custom.
+    default: true
```

Comment:
```suggestion
```

Let's remove the default: true because it's confusing. It meant that if you create the annotation, "true" would be the obvious default, but now the description already states that if it's missing, "false" is assumed.